commit 238221f3795e0664a67e955443ac6d5e2f5b1a03
Author: Christian Boltz <apparmor@cboltz.de>
Date:   Fri Jul 18 23:00:42 2025 +0200

    abstractions/X: allow reading /usr/share/xkeyboard-config-*/
    
    /usr/share/X11/xkb/ was moved to /usr/share/xkeyboard-config-2/ in
    xkeyboard-config 2.45, see
    https://gitlab.freedesktop.org/xkeyboard-config/xkeyboard-config/-/blob/master/ChangeLog.md?ref_type=heads#breaking-changes-2
    
    The old location was covered by the /usr/share/X11/** rule.
    
    Fixes: https://bugzilla.opensuse.org/show_bug.cgi?id=1246743

diff --git a/profiles/apparmor.d/abstractions/X b/profiles/apparmor.d/abstractions/X
index 48a5a5cbe..32aac7bd7 100644
--- a/profiles/apparmor.d/abstractions/X
+++ b/profiles/apparmor.d/abstractions/X
@@ -48,6 +48,8 @@
   /usr/share/X11/**               r,
   /usr/X11R6/**.so*               mr,
 
+  /usr/share/xkeyboard-config-[2-9]/** r,
+
   # EGL
   /usr/lib/@{multiarch}/egl/*.so* mr,
 
